.whatsnew.json

```
@@ -7,6 +7,10 @@
         "relativeLinkPrefix": "/dotnet/"
     },
     "areas": [
+        {
+            "name": ".",
+            "heading": "Miscellaneous"
```

Let's make this more clear that it's the hub area:

```suggestion
            "heading": "Home"
```